content/hi/docs/reference/glossary/container-env-variables.md

```diff
@@ -0,0 +1,16 @@
+---
+title: कंटेनर पर्यावरण चर (Container Environment Variables)
+id: container-env-variables
+date: 2018-04-12
+full_link: /docs/concepts/containers/container-environment/
+short_description: >
+  कंटेनर वातावरणीय वेरिएबल्स नाम=मान जोड़े होते हैं जो पॉड में चल रहे कंटेनरों को उपयोगी जानकारी प्रदान करते हैं
+aka: 
+tags:
+- fundamental
+---
+कंटेनर वातावरणीय चर नाम=मान जोड़े होते हैं जो {{< glossary_tooltip text="पॉड" term_id="pod" >}} में चल रहे कंटेनरों को उपयोगी जानकारी प्रदान करते हैं
+
+<!--more-->
+
+कंटेनर वातावरणीय चर वह जानकारी प्रदान करते हैं जो चल रहे कंटेनरीकृत अनुप्रयोगों के लिए आवश्यक होती है, साथ ही महत्वपूर्ण संसाधनों के बारे में जानकारी {{< glossary_tooltip text="कंटेनरों" term_id="container" >}} को भी प्रदान करती है। उदाहरण के लिए, फ़ाइल सिस्टम का विवरण, स्वयं कंटेनर के बारे में जानकारी, और अन्य क्लस्टर संसाधन जैसे कि सेवा अंत बिंदु।
```

Comment:
कंटेनर एनवायरनमेंट वेरिएबल्स उन जानकारी को प्रदान करते हैं जो चल रही कंटेनराइज्ड एप्लिकेशनों के लिए आवश्यक होती है, साथ ही महत्वपूर्ण संसाधनों की जानकारी भी {{< glossary_tooltip text="कंटेनरों" term_id="container" >}}  को देती हैं। उदाहरण के लिए, फाइल सिस्टम का विवरण, स्वयं कंटेनर के बारे में जानकारी, और अन्य क्लस्टर संसाधन जैसे सर्विस एंडपॉइंट्स।

Reply:
Thank you for the suggestions!! I have made all the necessary changes into it. Thanks!